waiter/src/waiter/scheduler.clj

```diff
@@ -323,6 +323,24 @@
   (merge existing-headers
          (retrieve-auth-headers service-id->password-fn service-id)))
 
+(defn classify-response-errors
+  "Return a corresponding error flag for this request, or nil if there was no error."
+  [{:keys [error] :as response}]
+  (cond
+    (some? error)
+    (cond
+      (instance? ConnectException error) :connect-exception
+      (instance? EOFException error) :hangup-exception
+      (instance? SocketTimeoutException error) :timeout-exception
+      (instance? TimeoutException error) :timeout-exception)
+    (utils/raven-proxy-response? response)
+    (when-let [raven-error (utils/raven-response-flags response)]
```

Comment:
```suggestion
    (when-let [reverse-proxy-error (utils/reverse-proxy-response-flags response)]
```

Reply:
Resolved this by consistently using raven as the name for the sidecar (see #1437).